src/pages/landing/forex-trading/components/_hero.tsx

```diff
@@ -142,20 +142,20 @@ const Hero = ({ title, content }: HeroProps) => {
 
     return (
         <div>
-            <Show.Desktop min_width="800">
```

Comment:
This refactor will not cause it to have the same behaviour as prod due to the 800px screen width config.